scene/gui/popup_menu.cpp

```diff
@@ -588,8 +588,8 @@ void PopupMenu::_input_from_window_internal(const Ref<InputEvent> &p_event) {
 	}
 
 	if (m.is_valid() && drag_to_press) {
-		BitField<MouseButtonMask> initial_button_mask = m->get_button_mask();
-		if (!initial_button_mask.has_flag(mouse_button_to_mask(MouseButton::LEFT)) && !initial_button_mask.has_flag(mouse_button_to_mask(MouseButton::RIGHT))) {
+		DisplayServer *ds = DisplayServer::get_singleton();
+		if (!ds->mouse_get_button_state().has_flag(mouse_button_to_mask(MouseButton::LEFT)) && !ds->mouse_get_button_state().has_flag(mouse_button_to_mask(MouseButton::LEFT))) {
```

Comment:
Copy paste mistake? It's checking `LEFT` twice.

Reply:
Oops. I need a coffee.